apps/application-system/api/infra/application-system-api.ts

```diff
@@ -154,20 +154,34 @@ export const serviceSetup = (services: {
       PAYMENT_ADDITION_CALLBACK_URL:
         '/k8s/application-system-api/PAYMENT_ADDITION_CALLBACK_URL',
       ARK_BASE_URL: '/k8s/application-system-api/ARK_BASE_URL',
+
       PARTY_APPLICATION_RVK_SOUTH_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_RVK_SOUTH_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_RVK_SOUTH_ASSIGNED_ADMINS',
       PARTY_APPLICATION_RVK_NORTH_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_RVK_NORTH_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_RVK_NORTH_ASSIGNED_ADMINS',
       PARTY_APPLICATION_SOUTH_WEST_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_SOUTH_WEST_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_SOUTH_WEST_ASSIGNED_ADMINS',
       PARTY_APPLICATION_NORTH_WEST_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_NORTH_WEST_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_NORTH_WEST_ASSIGNED_ADMINS',
       PARTY_APPLICATION_NORTH_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_NORTH_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_NORTH_ASSIGNED_ADMINS',
       PARTY_APPLICATION_SOUTH_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_SOUTH_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_SOUTH_ASSIGNED_ADMINS',
       PARTY_LETTER_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_LETTER_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_LETTER_ASSIGNED_ADMINS',
+
+      PARTY_APPLICATION_RVK_SOUTH_ADMIN_EMAIL:
```

Comment:
how often are these gonna change 😅?

Reply:
Once when we insert the correct values then every 4 years.
This is a temporary solution some of these variables will be replaced with the delegation system.
Others will be moved to a admin system that should be part of the next iteration of the system